.github/actions/release-candidate-ready-for-qa-v1/README.md

```diff
@@ -0,0 +1,34 @@
+# release-candidate-ready-for-qa-v1
+
+A GitHub Action to prepare already created a release candidate for QA.
+
+## Inputs
+
+| Name            | Required | Description                                                | Default            |
+| --------------- | -------- | ---------------------------------------------------------- | ------------------ |
+| `sha-rc`        | Yes      | SHA of the release commit from the release branch          | NA                 |
+| `slack-webhook` | Yes      | A Slack channel webhook URL where the message will be sent | NA                 |
+| `owner`         | No       | An owner of the repository                                 | $GITHUB_REPOSITORY |
+| `repo`          | No       | A repository name                                          | $GITHUB_REPOSITORY |
+
+## Example usage
+
+```yaml
+name: Release candidate ready for qa
+
+jobs:
+  release-candidate:
+    runs-on: ubuntu-latest
+    timeout-minutes: 2
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dequelabs/axe-api-team-public/.github/actions/release-candidate-ready-for-qa-v1-v1@main
+        with:
+          sha-rc: 4b632c61
```

Comment:
This shouldn't be an input, it should be calculated using the `release` branch sha.

```
git checkout release
git rev-parse --short HEAD
```

Reply:
a branch name should not be hardcoded. A workflow file should take responsibility for the branch name where this action is called